src/engine/enginemaster.h

```diff
@@ -291,6 +292,8 @@ class EngineMaster : public QObject, public AudioSource {
     CSAMPLE* m_pHead;
     CSAMPLE* m_pTalkover;
 
+    CSAMPLE** m_ppSidechain; // points to master or to talkover buffer
```

Comment:
please expand this comment.  What is the purpose of this double pointer?  Why not just use a single pointer?  We already have m_pEngineSideChain so this name needs to be different.